iml-agent/src/action_plugins/stratagem/server.rs

```diff
@@ -357,6 +373,61 @@ pub async fn trigger_scan(
     Ok((tmp_dir, x, mailbox_files))
 }
 
+/// Triggers a scan with Stratagem.
+/// This will only trigger a scan and return a triple of `(String, StratagemResult, MailboxFiles)`
+///
+/// It will *not* stream data for processing
+pub async fn trigger_scan_single(
+    _data: StratagemConfig,
+) -> Result<(String, StratagemResult, MailboxFiles), ImlAgentError> {
+    let id = Uuid::new_v4().to_hyphenated().to_string();
+
```

Comment:
What is this function for?  Testing?

Reply:
hangover from development.  It's been removed.